src/zigSetup.ts

```diff
@@ -37,8 +36,10 @@ async function getVersions(): Promise<ZigVersion[]> {
     })).data;
     const indexJson = JSON.parse(tarball);
     const result: ZigVersion[] = [];
-    for (let [key, value] of Object.entries(indexJson)) {
-        if (key == "master") key = "nightly";
+    for (const entry of Object.entries(indexJson)) {
+        let [key] = entry;
+        const [, value] = entry;
+        if (key === "master") {key = "nightly";}
```

Comment:
What's going on here?

Reply:
The value from entry isn't reassigned, so I had to make the key a variable a `let` variable and the value a `const` variable.

Reply:
Can the lint be overriden?

Reply:
Yes it can, I would make this changes.

Reply:
Thanks!